### ISSUE TITLE
Using array syntax on checkbox options results in duplicate key warning

### ISSUE THREAD

**Issue by aaronrosenthal** [wearebraid/vue-formulate#244]
**Describe the bug**
When using the array syntax for checkbox options, Vue throws a duplicate key warning to the console:

```
[Vue warn]: Duplicate keys detected: 'formulate-global-1'. This may cause an update error.


found in



---> <FormulateInputGroup>

       <FormulateInput>

         <Reproduction> at /src/components/Reproduction.vue

           <App> at /src/App.vue

             <Root>
```

**To Reproduce**
Use array syntax for checkbox options.

**Reproduction**
https://codesandbox.io/s/vue-formulate-reproduction-template-forked-gst8p

**Expected behavior**
Array syntax should behave just like the object syntax for checkbox options.

**Comment by justin-schroeder** [comment]
Hmm, youre correct that they shouldn't do that. If you provide an `id` in your array it should resolve, but ill mark this as a bug since it should be saling your array.

**Comment by justin-schroeder** [comment]
Closing this as it's fixed in `release/2.4.4` (to be released soon)
